backend/src/exceptions/auth-error/auth-error.ts

```diff
@@ -0,0 +1,18 @@
+import { HttpError } from '~/exceptions/exceptions';
+import {
+  CustomExceptionName,
+  ValidationMessage,
+  HttpCode,
+} from '~/common/enums/enums';
+
+class AuthError extends HttpError {
+  constructor({
+    message = ValidationMessage.EMAIL_ALREADY_EXISTS,
```

Comment:
I think there should another default message

Reply:
agred

Reply:
It was done according to the Mini Project
I guess it should be Internal Server Error by default, am I right?

Reply:
lets avoid default messages for services erros